.eslintrc

```diff
@@ -0,0 +1,78 @@
+{
+    "env": {
+        "commonjs": true,
+        "node": true,
+        "es2021": true
```

Comment:
As stated in [here](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments), putting es2021: true here enables ecmaVersion:12, likewise es6 enables ecmaVersion: 6. So, I think no need to write ecmaVersion explicitly. Btw, should we use es2021 and why? I thought these ecma versions are backward compatible and used the latest one.

Reply:
I'd prefer to keep using ES6 syntax, as Node.js 10 and 12 do not support all ES2021 features: https://node.green/#ES2020. The same consideration applies to ES2017+

Reply:
Actually, by setting ecmaVersion to 8 we were using es2017 instead of es6 before this pr. Setting es6: true makes ecmaVersion:7, then we make it 8 by reassigning it. For example async is added in es2017 which we used in tests, code samples, scripts. 

This auto setting is explained at https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments

Thus, setting es2017: true which sets ecmaVersion to 8 automatically.

Reply:
Makes sense. Let's use `es2017: true` then.